### ISSUE TITLE
[BUG] `module "rft_all_typing" exports "Foo" in the "__all__", but not defined it`

### ISSUE THREAD

**Issue by goraje** [dashingsoft/pyarmor#1285]
```
$ pyarmor gen --enable-rft rft_all_typing.py
INFO     Python 3.10.11
INFO     Pyarmor 8.2.1 (group), 006005, R****t
INFO     Platform darwin.aarch64
```

Pyarmor with rftmode enabled fails to obfuscate `__all__` for the following code:

```python
# rft_all_typing.py

__all__ = ["Foo"]

Foo: list[str] = [
    "foo1", "foo2",
]
```
with the following error:
```
ERROR    module "rft_all_typing" exports "Foo" in the "__all__", but not defined it
ERROR    invalid module "rft_all_typing"
```

Removing of type hints for `Foo` seems to fix the problem, which means that the same command runs perfectly fine for this code:
```
# rft_all_typing.py

__all__ = ["Foo"]

Foo = [
    "foo1", "foo2",
]
```

**Comment by jondy** [comment]
Got it, I'll check it.

**Comment by jondy** [comment]
Reproduced, because `ast.AnnAssign` is ignored, it will be fixed in next release.

**Comment by jondy** [comment]
Fixed it in v8.2.3
